src/policy/markcompactspace.rs

```diff
@@ -251,7 +251,12 @@ impl<VM: VMBinding> MarkCompactSpace<VM> {
             queue.enqueue(object);
         }
 
-        Self::get_header_forwarding_pointer(object)
+        let result = Self::get_header_forwarding_pointer(object);
+        assert!(
```

Comment:
Probably use `debug_assert!` instead. This is hot code for object tracing.

Reply:
Done.